driver-core/src/test/java/com/datastax/driver/core/AbstractReconnectionHandlerTest.java

```diff
@@ -330,7 +330,7 @@ protected void onReconnection(Connection connection) {
     private void waitForCompletion() {
         executor.shutdown();
         try {
-            boolean shutdown = executor.awaitTermination(10, TimeUnit.SECONDS);
+            boolean shutdown = executor.awaitTermination(30, TimeUnit.SECONDS);
```

Comment:
I've noticed this too (on other branches not just this one).  Would you be averse to moving these tests from 'unit' to 'short'?